src/pages/Auth/Register.tsx

```diff
@@ -1,107 +1,16 @@
-import { yupResolver } from "@hookform/resolvers/yup";
-import React, { useEffect, useState } from "react";
-import { useForm } from "react-hook-form";
-import { Link, useNavigate } from "react-router-dom";
-import * as Yup from "yup";
-import { useAuth } from "../../AuthContext";
-import FormError from "./FormError";
+import React from "react";
+import { useNavigate } from "react-router-dom";
 
-interface FormValues {
-  name: string;
-  email: string;
-  password: string;
-  confirmPassword: string;
-}
-
-const schema = Yup.object().shape({
-  name: Yup.string().required("Name is required"),
-  email: Yup.string()
-    .email("Invalid email address")
-    .required("Email is required"),
-  password: Yup.string()
-    .min(5, "Password must be at least 5 characters")
-    .required("Password is required"),
-  confirmPassword: Yup.string()
-    .oneOf([Yup.ref("password")], "Passwords do not match")
-    .required("Confirm password is required"),
-});
-
-const Register: React.FC = () => {
-  const { registerUser, currentUser } = useAuth();
+const RegisterType: React.FC = () => {
```

Comment:
may add some logic here where if a user is already signed in, then they should be redirected to the home page